src/Config/Converters/GraphQLRuntimeOptionsConverterFactory.cs

```diff
@@ -132,7 +132,13 @@ internal GraphQLRuntimeOptionsConverter(bool replaceEnvVar)
                             }
                             else if (reader.TokenType is JsonTokenType.Number)
                             {
-                                graphQLRuntimeOptions = graphQLRuntimeOptions with { DepthLimit = reader.GetInt32(), UserProvidedDepthLimit = true };
+                                int depthLimit = reader.GetInt32();
```

Comment:
try/catch?

Reply:
what happens when value in config (if manually set) is > Int32.Max?

Reply:
thanks for flagging. Added Try/Catch

Reply:
There have been no changes to this section of code.

Reply:
ignore, my github codespaces isn't updating correctly.